### PR TITLE
Create bookmarkcommand, however the bookmarked boolean of the stock i…

### LINKED ISSUE #26: As a tech savvy warehouse manager, I want to be able to bookmark certain items in the warehouse, so I can access and augment their information easily.

### PR DESCRIPTION
**Pull request by LimZeWeiDennis**
…s not stored. Will look into it again

### CODE REVIEW

File: src/main/java/seedu/stock/model/stock/Stock.java [hunk 1]
```diff
@@ -159,7 +183,8 @@ public boolean isSameStock(Stock otherStock) {
         return otherStock != null
                 && otherStock.getName().equals(getName())
                 && otherStock.getSerialNumber().equals(getSerialNumber())
-                && otherStock.getSource().equals(getSource());
+                && otherStock.getSource().equals(getSource())
+                && (otherStock.getBookmarked() == (getBookmarked()));
```
Comment (amylzting) [inline src/main/java/seedu/stock/model/stock/Stock.java]: Probably could remove this line? As we defined same stock as having same name, serial number and source.

File: src/main/java/seedu/stock/storage/JsonAdaptedStock.java [hunk 2]
```diff
@@ -119,7 +123,8 @@ public Stock toModelType() throws IllegalValueException {
                 modelNotesList.add(modelNote);
             }
 
-            return new Stock(modelName, modelSerialNumber, modelSource, modelQuantity, modelLocation, modelNotesList);
+            return new Stock(modelName, modelSerialNumber, modelSource, modelQuantity, modelLocation,
+                    modelNotesList, isBookmarked);
         }
 
         return new Stock(modelName, modelSerialNumber, modelSource, modelQuantity, modelLocation);
```
Comment (amylzting) [inline src/main/java/seedu/stock/storage/JsonAdaptedStock.java:130]: This part should be the reason why the stock is not updated. Should put the bookmark here as well. The bookmark above is for stock with notes.

**Review by amylzting — COMMENTED** [review]
There are some merge conflicts to resolve!

File: src/main/java/seedu/stock/model/stock/Stock.java (hunk 1 above)
Comment (LimZeWeiDennis) [reply src/main/java/seedu/stock/model/stock/Stock.java]: Yes, i was trying out if it affected and i forgot to remove it. Thanks!

**Comment by kaichaoang** [comment]
Do review the merge conflicts @LimZeWeiDennis under docs/DeveloperGuide.md!

**Review by AudreyFelicio — CHANGES_REQUESTED** [review]
The overall code looks good. Maybe some adjustments on help window initial size. I think with the new commands added it will become very long. Please resolve the merge conflict.

**Review by Ashley-Lau — CHANGES_REQUESTED** [review]
Looks good but bookmark remove and adding bookmark command to suggested command seems to be missing.

**Pull request closed without merging**